tests/UI/pages/BO/orders/add.js

```diff
@@ -191,6 +202,74 @@ class AddOrder extends BOBasePage {
     return page.frame({url: new RegExp(`sell/customers/${customerID}/view`, 'gmi')});
   }
 
+  /**
+   * Close iframe
+   * @param page {Page} Browser tab
+   * @returns {Promise<boolean>}
+   */
+  async closeIframe(page) {
+    await this.waitForSelectorAndClick(page, this.closeFancyBoxIframe);
+
+    return this.elementNotVisible(page, this.iframe, 3000);
+  }
+
+  /* Carts table methods */
+
+  /**
+   * Get text when carts table is empty
+   * @param page {Page} Browser tab
+   * @returns {Promise<string>}
+   */
+  async getTextWhenCartsTableIsEmpty(page) {
+    await page.waitForTimeout(2000);
```

Comment:
is it possible to remove this ?

Reply:
We added the timeout because the load selector is the same with empty table selector, then we should wait to display the text of empty table.